rasa/core/policies/ensemble.py

```diff
@@ -539,22 +496,6 @@ def is_not_in_training_data(
         # also check if confidence is 0, than it cannot be count as prediction
         return not is_memorized or max_confidence == 0.0
 
-    @staticmethod
-    def _is_not_mapping_policy(
-        policy_name: Text, max_confidence: Optional[float] = None
-    ) -> bool:
-        from rasa.core.policies.mapping_policy import MappingPolicy
-
-        is_mapping = policy_name.endswith("_" + MappingPolicy.__name__)
-        # also check if confidence is 0, than it cannot be count as prediction
```

Comment:
Is it ok that we no longer check this?

Reply:
As far as I can tell yes. Its only being used [here](https://github.com/RasaHQ/rasa/blob/08bf76bd70cc58784556e69c60b03570149f1119/rasa/core/policies/ensemble.py#L616-L621) as a condition in an if statement:
```python
if form_confidence is not None and self._is_not_mapping_policy(
    best_policy_name, best_confidence[0]
):
    # if mapping didn't win, check form policy predictions
    if form_confidence > best_confidence:
        best_policy_name = form_policy_name
```

Because we are also removing `FormPolicy`, `form_confidence` will always be `None` so this if condition can never evaluate to `True` anymore.